runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -211,6 +211,21 @@ doubleMaxMinHelper(TR::Node *node, TR::CodeGenerator *cg, bool isMaxOp)
    return node->getRegister();
    }
 
+static TR_OpaqueMethodBlock *
+getMethodFromBCInfo(
```

Comment:
There doesn't seem to be anything Z specific about this utility function. Does x86 / Power have a similar thing? Can we move it to a common place and share the implementation across codegens?

Reply:
This function needs Doxygen documentation as well.

Reply:
Refactored here: b8718b1